operantanalysis/scripts/CRF.py

```diff
@@ -0,0 +1,53 @@
+from operantanalysis import load_file, extract_info_from_file, reward_retrieval, lever_pressing, lever_press_latency
+import glob
+from tkinter import filedialog
+from tkinter import *  # noqa
+import pandas as pd
+import matplotlib
+matplotlib.use("TkAgg")
+from matplotlib import pyplot as plt  # noqa
+
+days = int(input("How many days would you like to analyze?"))
+column_list = ['Subject', 'Day', 'Dippers', 'Dippers Retrieved', 'Retrieval Latency',
+               'Lever Presses', 'Lever Press Latency']
+df = pd.DataFrame(columns=column_list)
+
+
+for i in range(days):
+    root = Tk()  # noqa
+    root.withdraw()
+    folder_selected = filedialog.askdirectory()
```

Comment:
why not to have this all as a function, in e.g. submodule `user_interact/crf.py` so you could later on compose even more elaborate interactive "workflows" by simply calling out to those functions?

Have a look also at "entrypoints" e.g. at https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html  or better come up with the `operantanalysis` command line tool, which could e.g. take an option `--cmd`, which would invoke those (and other) interactive functions. e.g. user (you) could say `operantanalysis --cmd CRF path1 path2 ...`, or without paths it would invoke your UI

Reply:
I added this as a function, and I am re-working the scripts- thank you!